Add OTLP Tracing support as a Zipkin trace input (#11287)
apache/skywalking: docs/en/setup/backend/otlp-trace.md

# OpenTelemetry Trace Format

> SkyWalking can receive traces from OpenTelemetry Trace and convert it into Zipkin Trace storage. 

Reviewer: ```suggestion
SkyWalking can receive traces from Traces in OTLP format and convert them to Zipkin Trace format eventually. 
```
